src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -329,10 +329,13 @@ bool VtolType::isRollExceeded()
 bool VtolType::isFrontTransitionTimeout()
 {
 	// check front transition timeout
-	if (getFrontTransitionTimeout()  > FLT_EPSILON && _common_vtol_mode == mode::TRANSITION_TO_FW) {
+	if (_common_vtol_mode == mode::TRANSITION_TO_FW) {
+		// when we use airspeed, we can timeout earlier if airspeed is not increasing fast enough
+		if (_param_fw_use_airspd.get() && _time_since_trans_start > getOpenLoopFrontTransitionTime()
+		    && _airspeed_validated->calibrated_airspeed_m_s < getBlendAirspeed()) {
+			return true;
 
-		if (_time_since_trans_start > getFrontTransitionTimeout()) {
-			// transition timeout occured, abort transition
+		} else if (_time_since_trans_start > getFrontTransitionTimeout()) {
```

Comment:
```suggestion
		} else if (getFrontTransitionTimeout() > FLT_EPSILON && _time_since_trans_start > getFrontTransitionTimeout()) {
```

Reply:
@sfuhrer The transition timeout should according to the param docs always be positive. I also think that one should always have a transition timeout enabled.
I adjusted the param documentation to remove the part which says that you can disable it.